### PR TITLE
Ft logout page

### PR DESCRIPTION
**Pull request by gerard1001**

### CODE REVIEW

**Comment by gerard1001** [comment]
Logout page

**Review by fab-ryan — APPROVED** [review]

**Review by UmuhireJessie — APPROVED** [review]

**Comment by Anguandia** [comment]
@gerard1001 , please give the PR a meaningful description as illustrated in the best practice doc and rebase

File: src/components/Logout/Logout.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+            <div className='icons'>
+              <BsIcons.BsBellFill className='icons-sub' /> 
+              <p>Notifications</p>
+            </div>
+
+            <div className='icons'>
+              <IoIcons.IoMdLogOut className='icons-sub' /> 
+              <p id='logout'>Logout</p>
+            </div>
```
Comment (furebo) [inline src/components/Logout/Logout.js:93]: Please add event to the logout

**Review by furebo — CHANGES_REQUESTED** [review]

**Review by aimelive — APPROVED** [review]

**Review by GilbertUp — APPROVED** [review]

**Comment by Anguandia** [comment]
This has already been implemented with login

**Pull request closed without merging**